Domains/1-Common/Profiles/Profiles.ecschema.xml

```diff
@@ -382,6 +383,7 @@
         <ECProperty propertyName="TopFlatWidth" typeName="double" displayLabel="Top Flat Width" description="Width of top flat."  kindOfQuantity="AECU:LENGTH_SHORT" />
         <ECProperty propertyName="BottomRibWidth" typeName="double" displayLabel="Bottom Rib Width" description="Width of bottom rib."  kindOfQuantity="AECU:LENGTH_SHORT" />
         <ECProperty propertyName="CoverageWidth" typeName="double" displayLabel="Coverage Width" description="Distance between top flat and bottom width."  kindOfQuantity="AECU:LENGTH_SHORT" />
+        <ECProperty propertyName="DiagonalWidth" typeName="double" displayLabel="Diagonal Width" description="Horizontal width of (a single) web diagonal ."  kindOfQuantity="AECU:LENGTH_SHORT" />
```

Comment:
Unnecessary space at the end of description.

Reply:
Thanks for noticing!